nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/FormActivity.java

```diff
@@ -11,7 +11,7 @@
  * {@link FormWithAppCompatActivity}
  */
 public abstract class FormActivity extends Activity {
-
+    private static final String MODEL_KW = "model";
```

Comment:
Not sure what the `KW` part stands for. What do you think of `MODEL_BUNDLE_KEY` as the constant name? Also `MODEL_KEY` works.

Also, let's prefix the value with `nd_` (e.g. `nd_model`) to prevent collisions in case the the user also persists data in the Bundle.